index.html

```diff
@@ -486,6 +535,46 @@ <h2>Pros and Cons</h2>
                 </p>
             </section>
         </section>
+        <section id=hybrid>
+            <h2>Hybrid solution</h2>
+            <p>
+                This approach considers the “ideal” solution of <a href="#extending-lang-string">extending `langString`</a> (taking care of the core problem in RDF) but adding some feasibility considerations. The goal is to avoid the necessity to <a href="#full-family">update all RDF related specifications</a> at the same time. This could be achieved as follows.
+            </p>
+
+            <ul>
+                <li>The core RDF 1.1 specification is updated, as described in <a href="#langString">the section above</a>.</li>
+                <li>New datatype(s) (as described <a href="datatype">above</a>) are defined, but with the following modifications:
+                    <ul>
+                        <li>
+                            these datatypes are defined to be deprecated soon, i.e., they are considered to be of a temporary usage only;
+                        </li>
+                        <li>
+                            the value space for the datatype(s) consists of the renewed language string tuples as defined in the (renewed) RDF semantics document&nbsp;[[rdf11-mt]];
+                        </li>
+                        <li>the upcoming versions of N-Triple, N-Quads, Turtle, and TriG serializations are <em>not</em> modified; i.e., the only way to express the renewed <code>langString</code> is to use the new datatype(s).</li>
```

Comment:
I was not suggesting that it would be the *only* way, only the *default* way. I would still like to be able to write in Turtle 1.2 or SPARQL 1.2  `"hello"@en^ltr` or may be `"hello"@en_ltr` (for the sake of regularity), instead of `"hello"^^i18n:en_ltr`, even though the latter would be the default for serializers (for backward compatibility).

Proposed rephrasing:

```suggestion
                        <li>in the upcoming versions of N-Triple, N-Quads, Turtle, and TriG, literals with a base direction would still be expressed using the <code>"value"^^i18n:XX_YY</code> syntax, but that syntax would be specially parsed as the corresponding (value, language-tag, direction) triple, <em>and the <code>rdf:langString</code> datatype</em>.</li>
```

For the sake of simpicity, I do not mention the possibility of other syntaxes, but I do not exclude it either.

Reply:
I formalized a bit differently, so check the latest version... but I believe I captured what you said (and agree with it!)